app/src/main/java/com/teamwss/websoso/ui/onboarding/OnboardingViewModel.kt

```diff
@@ -130,8 +130,14 @@ class OnboardingViewModel @Inject constructor(
         )
     }
 
+    fun updateNicknameInputType(nicknameInputType: NicknameInputType) {
+        updateOnBoardingFirstUiState(nicknameInputType, "")
+    }
+
     fun clearInputNickname() {
-        currentNicknameInput.value = ""
+        if(onboardingFirstUiState.value?.nicknameInputType != NicknameInputType.COMPLETE){
```

Comment:
c: 
```suggestion
        if(onboardingFirstUiState.value?.nicknameInputType != NicknameInputType.COMPLETE) {
```
여기 코드 포맷팅이 안 되어있는 듯욥